theme/src/components/Footer.tsx

```diff
@@ -44,8 +46,11 @@ const FOOTER_LINKS: FooterItem[] = [
   },
 ];
 
-const COMMON_STYLES =
-  'whitespace-nowrap text-xs screen-450:text-sm text-white mr-6 last:mr-0';
+const COMMON_STYLES = clsx(
+  'text-xs screen-450:text-sm text-white',
+  'whitespace-nowrap mr-6 last:mr-0',
+);
```

Comment:
sorry if there wasn't autocomplete for this, i need to add a Taiwind autocomplete pattern for this use case 😭